src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_modular.py

```diff
@@ -285,7 +281,7 @@ def prepare_ip_adapter_image_embeds(
     def __call__(self, pipeline, state: PipelineState) -> PipelineState:
         data = self.get_block_state(state)
 
-        data.do_classifier_free_guidance = data.guidance_scale > 1.0
+        data.do_classifier_free_guidance = pipeline.guider.num_conditions > 1
```

Comment:
`do_classifier_free_guidance`: is this still a good name (? we have option to change to something else with modular I think